src/core/adapter/devhub-contract.jsx

```diff
@@ -62,13 +62,6 @@ function updateCommunityBoard({ handle, board }) {
   });
 }
 
-function updateCommunityGithub({ handle, github }) {
-  return Near.call("${REPL_DEVHUB_CONTRACT}", "update_community_github", {
```

Comment:
`update_community_board` can also be removed since we now use `update_addon`.

Reply:
yeah I will remove the unused code after updating kanban board as well